src/main/java/org/folio/dew/service/ModuleTenantService.java

```diff
@@ -0,0 +1,70 @@
+package org.folio.dew.service;
+
+import static org.folio.dew.utils.Constants.EUREKA_PLATFORM;
+
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.dew.client.EurekaProxyTenantsClient;
+import org.folio.dew.client.OkapiClient;
+import org.folio.dew.domain.bean.ModuleForTenant;
+import org.folio.dew.error.NotFoundException;
+import org.folio.spring.FolioExecutionContext;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Optional;
+
+
+@Log4j2
+@Service
+@RequiredArgsConstructor
+public class ModuleTenantService {
+
+  private static final String MODULE_NOT_FOUND_ERROR = "Module id not found for name: ";
```

Comment:
This can be changed to pattern  ""Module id not found for name: %s"".

Reply:
Create a separate constant - MOD_USERS_NOT_FOUND_ERROR. Thansk.